torch/csrc/jit/tensorexpr/stmt.h

```diff
@@ -433,6 +433,14 @@ class TORCH_API Cond : public StmtNode<Cond> {
     }
   }
 
+  Cond* cloneWithNewBodies(Stmt* true_stmt, Stmt* false_stmt) {
```

Comment:
nit: why not have the same name as the function below, `cloneWithNewBody`?

Reply:
Because it has two bodies! (true_stmt and false_stmt) whereas the one below only has a single body.

Reply:
I was thinking about having the same name and have the two body version as an overload. Never mind.